modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -230,6 +231,7 @@ const SidebarPanelInfoView = ({
 					)}
 
 					<SidebarPanelInfoCollapsibleSection
+						expanded={false}
```

Comment:
Is this prop needed? It's set false by default. Do you prefer to declare it for readability or something like that?